src/main.py

```diff
@@ -109,14 +109,14 @@ def main(args):
 
     # start the nodes with operations runner based on what's specified
     flags = {'runOps': args.run_test_ops,
-             'failure_rate': args.failure_rate,
-             'recover_rate': args.recover_rate}
+             'heartbeat_delay': args.heartbeat_delay,
```

Comment:
Remove these arguments. I don't see the need to make these cmd line args.

Reply:
Remove both heartbeat_delay etc. two arguments